airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_is.properties

```diff
@@ -702,7 +702,7 @@ helppopup.ldap.text=<p> Notendur geta a\u00F0 au\u00F0kenna me\u00F0 utana\u00F0
 helppopup.ldapurl.title=LDAP URL
 helppopup.ldapurl.text=<p> sl\u00F3\u00F0 \u00E1 LDAP mi\u00F0lara. Samskiptareglan ver\u00F0ur anna\u00F0hvort <em> LDAP ://</ em> e\u00F0a <em> ldaps ://</ \u00ED> (fyrir LDAP yfir SSL). Sj\u00E1 <a href="http://java.sun.com/products/jndi/tutorial/ldap/misc/url.html" target="_blank"> h\u00E9r </ a> fyrir n\u00E1kv\u00E6mari l\u00FDsingu. </ p>
 helppopup.ldapsearchfilter.title=LDAP Search Filter
-helppopup.ldapsearchfilter.text=<p> s\u00EDa tj\u00E1ning notu\u00F0 sem notandinn leitar. \u00DEetta er LDAP leit s\u00EDu (eins og skilgreint \u00ED <a href="http://www.ietf.org/rfc/rfc2254.txt" target="_blank"> RFC 2254 </ a>). The mynstur " '(0')" komi username, til d\u00E6mis: <ul> <li> (uid = '(0')) - \u00FEetta v\u00E6ri a\u00F0 leita a\u00F0 notandanafn samsv\u00F6run \u00E1 UID eiginleiki. </ li> <li> (sAMAccountName = '(0')) - oftast notu\u00F0 fyrir sta\u00F0festing \u00E1 Microsoft Active Directory. </ li> </ ul> </ p>
+helppopup.ldapsearchfilter.text=<p> s\u00EDa tj\u00E1ning notu\u00F0 sem notandinn leitar. \u00DEetta er LDAP leit s\u00EDu (eins og skilgreint \u00ED <a href="http://www.ietf.org/rfc/rfc2254.txt" target="_blank"> RFC 2254 </ a>). The mynstur "'{0}'" komi username, til d\u00E6mis: <ul> <li> (uid = '{0}') - \u00FEetta v\u00E6ri a\u00F0 leita a\u00F0 notandanafn samsv\u00F6run \u00E1 UID eiginleiki. </ li> <li> (sAMAccountName = '{0}') - oftast notu\u00F0 fyrir sta\u00F0festing \u00E1 Microsoft Active Directory. </ li> </ ul> </ p>
```

Comment:
This one file was different than all the rest, it had parens instead of curly braces.